### PR TITLE
Refactor and top/bottom shadows refactoring

### PR DESCRIPTION
**Pull request by soswow**
Simplify code and make it work with flexible container. + Implement top/bottom shadows. + readme update

### CODE REVIEW

File: example/app.tsx [hunk 1]
```diff
@@ -5,17 +5,26 @@ import {Item, AppWrapper, AppHeader, AppFooter} from './styled';
 
 export default class App extends Component {
   render() {
-    const items = Array(20).fill().map((value, key) => 
-      <Item key={key}>{key}</Item>
-    );
+    const items = [];
```
Comment (soswow) [inline example/app.tsx:8]: I was testing it in IE11. Flex doesn't work there, but I couldn't see a thing because of this JS.

Comment (zzarcon) [reply example/app.tsx:8]: Good call, I wasn't bothering about having IE support, but if you fixed it awesome!

**Review by zzarcon — APPROVED** [review]

**Comment by zzarcon** [comment]
This looks amazing to me! Thanks for the great job 💃

File: example/app.tsx (hunk 1 above)
Comment (soswow) [reply example/app.tsx:8]: It does work in Edge, Chrome, Safari, Firefox. IE11 doesn't support Flex, but it shows content, which is good.

**Pull request merged**